### ISSUE TITLE
Submit a pull request

### ISSUE THREAD

**Issue by willrlin** [google-org/sandbox-detroit#3]
## Expected Behavior
Have our contributors get used to submitting pull requests.

**Comment by anders-schneider** [comment]
Mic check one two (just testing writing an update on this issue :)

**Comment by willrlin** [comment]
Good test! I should check my notification settings now.

**Comment by anders-schneider** [comment]
:) thanks.

So far my attempt at creating a pull request has been unsuccessful - I'm following https://opensource.com/article/19/7/create-pull-request-github but `git push -u origin andys_branch` fails because after I enter my username and password ("remote: Invalid username or password."). I'm sure I've misconfigured some authorization thing, I'll come back to this later

**Comment by willrlin** [comment]
Try using the GitHub desktop app, it definitely makes things easier.

**Comment by willrlin** [comment]
I noticed that when you submitted your pull request it included all of the history of the 3 commits you did. I wonder if it's easy in the GUI to squash the commits into 1.

**Comment by anders-schneider** [comment]
Yeah I remember there was an option at one point (not sure if in the desktop app or the web UI) to either squash into a single commit or keep the individual commits. (I kept them as individuals because I wanted to see what it would look like.) I'll create another pull request and squash the multiple commits!

**Comment by anders-schneider** [comment]
After poking around a bit, it looks like there's no way to link an issue to a pull request without making it so that merging the pull request causes the issue to be closed?

**Comment by anders-schneider** [comment]
Okay so it looks like it is possible to reference an issue without closing it by putting the issue number in the squashing commit description.

**Comment by anders-schneider** [comment]
Actually, that "Test pull request that squashes multiple commits into one" PR isn't listed as a "Linked pull request" :/ so I guess it only appears in the comment thread above: "anders-schneider added a commit that referenced this issue"

**Comment by willrlin** [comment]
Good to know! Also, it seems like it added a single commit to this issue, but if you look at the actual pull request you can still see all the individual commits that occurred before you merged it.
